rust.spec

```
@@ -220,7 +222,6 @@ test -f '%{local_rust_root}/bin/rustc'
 
 %setup -q -n %{rustc_package}
```

Can be simplified a bit:
```suggestion
%autosetup -p1 -n %{rustc_package}
```